test/nocks.ts

```diff
@@ -31,25 +31,38 @@ export function oauth2<T extends {} = {}>(validator?: (body: T) => boolean):
       });
 }
 
-export function projectId(reply: () => string) {
+export function projectId(status: number|(() => string), reply?: () => string) {
+  if (typeof status === 'function') {
+    reply = status;
+    status = 200;
+  }
   return nock('http://metadata.google.internal')
       .get('/computeMetadata/v1/project/project-id')
       .once()
-      .reply(reply);
+      .reply(200, reply, {'Metadata-Flavor': 'Google'});
```

Comment:
I think you should use `status` instead of 200, right? Same for similar pieces of code below.

Reply:
Done